.github/release.yaml

```diff
@@ -0,0 +1,18 @@
+# This file determines how the auto-generated Release Notes in GitHub releases are structured.
+
+changelog:
+  exclude:
+    labels:
+      - no-changelog
+    authors:
+      - dependabot[bot]
+  categories:
+    - title: Bugs
+      labels:
+        - bug
+    - title: New Features & Improvements
+      labels:
+        - "*"
```

Comment:
Does this mean all labels including or excluding `bug` and `documentation`?

Reply:
I would assume so, but to be honest I don't know exactly - documentation is sparse and I wasn't able to test this on my fork, so I guess we'll have to run a test once the PR is merged.